public/provisioning.php

```diff
@@ -59,6 +59,11 @@
     //Add token2 variable
     $scope_data['tok2'] = \Tancredi\Entity\TokenManager::getToken2($id);
 
+    // Add provisioning_complete variable
+    if ($token === $scope_data['tok2']) {
+        $scope_data['provisioning_complete'] = TRUE;
```

Comment:
```suggestion
        $scope_data['provisioning_complete'] = '1';
```

Add also the "else" case:

```
        $scope_data['provisioning_complete'] = '';
```